tests/test_config.py

```diff
@@ -32,6 +32,7 @@ def test_from_config(self):
             eq_(Configuration.DEFAULT_FACET, facet_config._default_facets)
 
         with temp_config() as config:
+            config[Configuration.POLICIES] = {}
```

Comment:
Unrelated change that has been annoying me for a while. I run tests with an extremely minimal configuration that doesn't define POLICIES at all, which is legal but which broke this test.